src/test/java/de/szut/lf8_project/repository/TestEmployeeRestRepository.java

```diff
@@ -0,0 +1,112 @@
+package de.szut.lf8_project.repository;
+
+import de.szut.lf8_project.common.JWT;
+import de.szut.lf8_project.common.Statuscode;
+import de.szut.lf8_project.domain.Employee.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.*;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.annotation.DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD;
+
+@ExtendWith(MockitoExtension.class)
+@DirtiesContext(classMode = AFTER_EACH_TEST_METHOD)
```

Comment:
@DisplayName("A EmployeeRestRepository")

Reply:
Dann sollten wir aber auch bei allen Tests
Displaynames mit "Should ..." haben.

Reply:
stimmt, oder oben in den Klassen-Displayname rein